cameraview/src/main/java/com/otaliastudios/cameraview/Camera1.java

```diff
@@ -606,14 +696,20 @@ boolean startVideo(@NonNull File videoFile) {
                 mMediaRecorder.start();
                 return true;
             } catch (Exception e) {
-                LOG.e("Error while starting MediaRecorder. Swallowing.", e);
+                CameraException cameraException =
```

Comment:
This is a buggy hardware component so it's cool to catch, but you don't need to throw the exception outside and crash, we are already recovering

Reply:
This is why I would distinguish between fatal errors as `CameraUnavailableException` and non-fatals as the one thrown here. This code refers to the issue in #58 and as stated there, I think some of the exceptions can't be prevent. In my opinion, there should be a smooth way to inform the user about the failed video capturing instead of just recovering the camera state silently.